src/sentry/ingest/transaction_clusterer/rules.py

```diff
@@ -73,7 +73,7 @@ def write(self, project: Project, rules: RuleSet) -> None:
 
 class CompositeRuleStore:
     #: Maximum number (non-negative integer) of rules to write to stores.
-    MERGE_MAX_RULES: int = 25
+    MERGE_MAX_RULES: int = 30
```

Comment:
I suspect that an increase of +5 will not bring a qualitative change to the number of orgs that bump into this limit, so maybe increasing by 2x will tell us more.

```suggestion
    MERGE_MAX_RULES: int = 50
```